packages/css-library/src/tokens/border.scss

```diff
@@ -0,0 +1,18 @@
+@use './_variables' as *;
+
+$tokens-border: (
+  '0': '0',
+  '1px': '1px solid',
+  '2px': '2px solid',
+  '3px': '3px solid',
+  '4px': '4px solid',
+  '5px': '5px solid',
+  '7px': '7px solid',
+  '10px': '10px solid',
+);
+
+$tokens-border-style: (
+  'solid': 'solid',
```

Comment:
Would it make sense to use Style Dictionary for things like this instead?

Reply:
We talked about this and decided that SD tokens aren't that helpful here at the moment.